regexes/device/mobiles.yml

```diff
@@ -1540,6 +1539,9 @@ Sony:
       model: 'Xperia X'
     - regex: '(?:Sony(?:Ericsson)?)?F5321'
       model: 'Xperia X Compact'
+    - regex: 'E5506'
+      model: 'Xperia C5 Ultra'
+      brand: 'Sony'
```

Comment:
no need to define the brand here, as `Sony` is the default. You can remove this line